packages/cli/src/commands/exchange/list.ts

```diff
@@ -25,17 +22,15 @@ export default class List extends BaseCommand {
 
     cli.action.start('Fetching exchange rates...')
 
-    const exchange = await Exchange(this.web3)
-    const dollarForGold = await exchange.methods
-      .getBuyTokenAmount(parsedFlags.amount as string, true)
-      .call()
-    const goldForDollar = await exchange.methods
-      .getBuyTokenAmount(parsedFlags.amount as string, false)
-      .call()
+    const exchange = await this.kit.contracts.getExchange()
+
+    const dollarForGold = await exchange.getBuyTokenAmount(parsedFlags.amount as string, true)
+
```

Comment:
Tiny style opinion nit: I'd remove this newline since the two lines around are related.